packages/sil-voting-app/src/routes/dashboard/components/Proposal/VotingPanel/Buttons.tsx

```diff
@@ -15,6 +15,7 @@ interface Props {
 
 const Buttons = ({ id, vote }: Props): JSX.Element => {
   const [currentVote, setCurrentVote] = useState(vote);
+  const [previousVote, setPreviousVote] = useState(currentVote);
```

Comment:
This could also be `= useState(currentVote);` like above, right? If you do that, it makes the source of the initial state more direct and you can swap the order of those lines easily

Reply:
Do you mean `const [previousVote, setPreviousVote] = useState(vote);`? If not I'm afraid I don't understand.

Reply:
I do